.travis.yml

```diff
@@ -57,25 +44,31 @@ matrix:
     - php: 7.1
       env:
         - DEPS=latest
-    - php: hhvm 
+    - php: 7.2
+      env:
+        - DEPS=lowest
+    - php: 7.2
+      env:
+        - DEPS=locked
+    - php: 7.2
+      env:
+        - DEPS=latest
+    - php: nightly
       env:
         - DEPS=lowest
-    - php: hhvm 
+    - php: nightly
       env:
         - DEPS=locked
-    - php: hhvm 
+    - php: nightly
       env:
         - DEPS=latest
   allow_failures:
-    - php: hhvm
-
-notifications:
-  irc: "irc.freenode.org#zftalk.dev"
-  email: false
+    - php: 7.2
+    - php: nightly
```

Comment:
Too early... Can be removed, see: https://github.com/zendframework/zend-expressive-hal/pull/9